openreview/venue/invitation.py

```diff
@@ -1037,6 +1037,32 @@ def set_official_comment_invitation(self):
         if comment_expdate:
             invitation.edit['invitation']['expdate'] = comment_expdate
 
+        if self.venue.ethics_review_stage:
+            invitation.edit['content']['noteReaders'] = {
+                'value': {
+                    'param': {
+                        'type': 'string[]', 'regex': f'{venue_id}/.*|everyone'
+                    }
+                }
+            }
+            invitation.content['comment_readers'] = {
+                'value': comment_stage.get_readers(self.venue, '{number}')
+            }
+            invitation.content['readers_delection'] = {
+                'value': comment_stage.reader_selection
+            }
+            comment_readers = ['${5/content/noteReaders/value}']
+            if comment_stage.reader_selection:
+                comment_readers = {
+                    'param': {
+                        'enum': ['${7/content/noteReaders/value}']
+                    }
+                }
+            invitation.edit['invitation']['edit']['note']['readers'] = comment_readers
+
+            invitation.edit['invitation']['invitees'].append(self.venue.get_ethics_reviewers_id('${3/content/noteNumber/value}'))
+            invitation.edit['invitation']['edit']['signatures']['param']['regex'] += '|' + self.venue.get_ethics_reviewers_id('${5/content/noteNumber/value}')
```

Comment:
@melisabok for comment invitees and signatures, I am always adding the SubmissionX/Ethics_Reviewers even when the paper is not flagged. These groups won't exist for not flagged papers so it shouldn't be an issue. Is this ok? It would be more complicated to have to do the same with invitees and sigantures as we are doing with readers (saving them in the invitation content and passing them when creating the paper comment invitations)

Reply:
ok, I think that's fine. 

I'm worried that a paper is flagged, assigned ethics reviewers and then un-flagged. But we un-flag a paper we are not doing anything in the process function, right?

Reply:
not yet. When we unflag it, I can expire the ethics review invitation and perhaps empty or delete the ethics reviewers group?

Reply:
Can we unflag a paper by deleting the `ethics_flag` field from the submission or should I do it by using Yes/No?

Reply:
I think with the value "No" is easier to unflag? what do you think?

If we remove the field then we need to remove the other one with the comment and from the UI is not so clear that deleting the data will unflag the paper